.gitignore

```diff
@@ -1 +1,4 @@
+# Rust build cache and outputs
 /target
+# Nix outputs
+/result
```

Comment:
While being there, I'd probably add `.direnv/` right away since a lot of people use direnv with nix, and I'd also suggest to provide a `.envrc` file with `use flake` so people who use direnv would be already setup.

Reply:
Fair point, added that in case someone might find it useful.